### ISSUE TITLE
Upload support

### ISSUE THREAD

**Issue by anytarseir67** [scrazzz/redgifs#4]
Are there currently any plans to add upload support to the lib?

**Comment by scrazzz** [comment]
Hey, thanks for opening an issue.

Yes, I had planned on adding it. I haven't gotten to implement it yet because I felt like I might waste my time on implementing something that a very small group of people would use it.

Since you asked for it. I'll see what I can do about it.

**Comment by scrazzz** [comment]
Sorry, I don't plan on adding this feature anytime soon.

**Comment by scrazzz** [comment]
On second thought, I feel like I need this feature too. So I'll add it back to planned.

**Comment by scrazzz** [comment]
Ok. So I tried requesting an API key to test out the uploading API endpoint but it looks like I have to go ahead and fill out a *google* form for it. I asked if there was an alternative way to do the form by email and they declined.

There was another user who tried requesting for an API key for an app they were making and one of the RedGIFs dev declined it as it was "not a use-case that would require a lot of people using it".

I wouldn't be surprised if I did not get an API key even if I filled out the form.
